src/modules/webhooks/get-interaction/1.0/module.json

```diff
@@ -1,40 +1,1053 @@
+    },
+    {
+      "name": "Source Details Person Person Id",
+      "value": "$.interaction.sourceDetails.person.personId",
+      "defaultValue": "sourceDetailsPersonPersonId"
+    },
+    {
+      "name": "Source Details Person Session Id",
+      "value": "$.interaction.sourceDetails.person.sessionId",
+      "defaultValue": "sourceDetailsPersonSessionId"
+    },
+    {
+      "name": "Source Details Person Phone Number",
+      "value": "$.interaction.sourceDetails.person.phoneNumber",
+      "defaultValue": "sourceDetailsPersonPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number",
+      "value": "$.interaction.sourceDetails.phoneNumber",
+      "defaultValue": "sourceDetailsPhoneNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Type",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.type",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Type",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberType",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberType"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country Calling Code",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.countryCallingCode",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountryCallingCode"
+    },
+    {
+      "name": "Source Details Phone Number Formatted National Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.nationalNumber",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNationalNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.number",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumber"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Country",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.country",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedCountry"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Area Code",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.areaCode",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsAreaCode"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Exchange",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.exchange",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsExchange"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Number Parts Last Four",
+      "value": "$.interaction.sourceDetails.phoneNumberFormatted.numberParts.lastFour",
+      "defaultValue": "sourceDetailsPhoneNumberFormattedNumberPartsLastFour"
+    },
+    {
+      "name": "Source Details Phone Number Formatted Formatted International",
```

Comment:
Lets make these like `Source Phone Number International Format` and adjust the defaultValue to similar. Same idea for national as well